### PR TITLE
docs-headless-guide: add headless app guide

### PR DESCRIPTION
**Pull request by sumitsaurabh927**

### CODE REVIEW

File: guides/headless-notification-center-guide.mdx [hunk 1]
```diff
@@ -0,0 +1,410 @@
...
+2. Click `Workflows` on the left sidebar of your Novu dashboard.
+3. Click the `Create Workflow` button on the top right:
+<Frame caption="Creating workflows is simple"><img src="/images/create-workflow.png" /> </Frame>
+Once you click the `create workflow` button, you’ll see a dropdown. Select `blank workflow` from the dropdown:
+<Frame caption="Select `blank workflow` from the dropdown"><img src="/images/blank-workflow.png" /> </Frame>
+You’ll now be taken to the workflow editor:
+<Frame caption="Workflow editor allows you to create and edit workflows"><img src="/images/workflow-editor.png" /> </Frame>
+Once here, you can add the channels you want to use for sending notifications and configure them. For this guide, we’ll use the `In-App` channel. 
+<Frame caption="The headless demo workflow that we'll be using"><img src="/images/SCR-20230825-pigg.png" /> </Frame>
+The `in-app` node allows you to further customize the notifications that will get sent, as per your need. In our case, it is a simple `description`, which will contain the text that will be entered from the front end. Here's the `in-app` node for your reference:
```
Comment (unicodeveloper) [inline guides/headless-notification-center-guide.mdx:35]: in-app should always be `In-App`

File: guides/headless-notification-center-guide.mdx [hunk 2]
```diff
@@ -0,0 +1,410 @@
...
+icon: 'circle-3'
+---
+
+## Introduction
+In this guide, you'll learn about how to use the headless version of our notification center. The headless version is useful for scenarios in which a lightweight solution for integrating notification functionality into a web app that is completely unstyled (so that you can style it as per your wish) is required.
+<br/>
+You can find the entire code ( front-end as well as back-end ) of this app [here.](https://github.com/novuhq/novu-headless-demo-app)
+
+## What is the headless notification center
+The headless version of Novu's notification library package provides users with a lightweight solution for integrating notification functionality into their web applications. With just the essential API methods, users can easily incorporate our notification system into any framework or vanilla JavaScript project, without being constrained by our default UI or dependencies.
```
Comment (unicodeveloper) [inline guides/headless-notification-center-guide.mdx:13]: `With just the essential...`

Please start it on another paragraph..

Comment (sumitsaurabh927) [reply guides/headless-notification-center-guide.mdx:13]: okay

File: guides/headless-notification-center-guide.mdx (hunk 1 above)
Comment (sumitsaurabh927) [reply guides/headless-notification-center-guide.mdx:37]: fixing

File: notification-center/client/headless/get-started.mdx [hunk 3]
```diff
@@ -128,3 +128,7 @@ headlessService.listenUnreadCountChange({
 [Explore Implementation of the Headless Library in an App](https://github.com/novuhq/novu-headless-demo-app)
 
 [Headless Demo](https://headlessdemo.vercel.app)
+
+<Tip>
+If you're ready to start using the Headless Notification Center, check out our guide [here!](/guides/headless-notification-center-guide)
```
Comment (unicodeveloper) [inline notification-center/client/headless/get-started.mdx:133]: no need for `here`. you can stop at guide.

File: sdks/headless-javascript-service.mdx [hunk 4]
```diff
@@ -8,3 +8,7 @@ description: "Novu’s headless library provides users with a lightweight, compl
 With just the essential API methods, users can easily incorporate our notification system into any framework or vanilla JavaScript project, without being constrained by our default UI or dependencies.
 
 Learn more about usage of the [Headless Services in detail.](/notification-center/client/headless/get-started)
+
+<Tip>
+If you're ready to start using the Headless Notification Center, check out our guide [here!](/guides/headless-notification-center-guide)
```
Comment (unicodeveloper) [inline sdks/headless-javascript-service.mdx:13]: no need for `here`. stop at guide

**Pull request merged**